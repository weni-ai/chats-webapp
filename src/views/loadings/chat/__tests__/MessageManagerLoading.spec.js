import { mount } from '@vue/test-utils';
import MessageManagerLoading from '../MessageManager.vue';
import { beforeEach, describe, expect } from 'vitest';

const createWrapper = () => {
  return mount(MessageManagerLoading);
};

describe('MessageManagerLoading.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = createWrapper();
  });

  it('to match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
