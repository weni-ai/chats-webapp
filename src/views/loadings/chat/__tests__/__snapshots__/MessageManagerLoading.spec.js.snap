// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MessageManagerLoading.vue > to match snapshot 1`] = `"<div data-v-63c78ce3="" class="message-manager-loading"><span data-v-63c78ce3=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-63c78ce3=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-63c78ce3=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div>"`;
